extensions/oidc/deployment/src/main/java/io/quarkus/oidc/deployment/OidcBuildStep.java

```diff
@@ -44,9 +43,11 @@ EnableAllSecurityServicesBuildItem security() {
         return new EnableAllSecurityServicesBuildItem();
     }
 
+    @SuppressWarnings("deprecation")
     @Record(ExecutionTime.RUNTIME_INIT)
     @BuildStep
-    public void setup(OidcConfig config, OidcRecorder recorder, VertxBuildItem vertxBuildItem,
+    public void setup(OidcConfig config, OidcRecorder recorder,
+            io.quarkus.vertx.core.deployment.InternalWebVertxBuildItem vertxBuildItem,
```

Comment:
It's minor but could we avoid using the FQCN here and just adding an import?

Reply:
@gsmet sure, I did it earlier to avoid multiple deprecation warnings